assets/js/mapsAPI.js

```diff
@@ -1,49 +1,72 @@
 function initialize() {
-  $.getJSON('https://githubschool.github.io/open-enrollment-classes-introduction-to-github/createMap.topojson', function (data) {
-    var map = new google.maps.Map(document.getElementById('map'), {
-      zoom: 2,
-      minZoom: 2,
-      center: new google.maps.LatLng(10, 15),
-      mapTypeId: 'terrain',
-      //disableDefaultUI: true,
-      mapTypeControl: false,
-      panControl: false,
-      //scaleControl: false,
-      scrollwheel: false,
-      streetViewControl: false,
-      //zoomControl: false,
-      //draggable: false,
-      styles: [{
-        featureType: 'administrative',
-        elementType: 'geometry',
-        stylers: [{visibility: 'off'}]
-      }, {
-        featureType: 'administrative.country',
-        stylers: [{visibility: 'off'}]
-      }, {
-        featureType: 'water',
-        elementType: 'labels',
-        stylers: [{visibility: 'off'}]
-      }, {
-        featureType: 'administrative',
-        elementType: 'labels',
-        stylers: [{visibility: 'off'}]
-      }]
-    });
+  $.getJSON(
+    'https://githubschool.github.io/open-enrollment-classes-introduction-to-github/createMap.topojson',
+    function(data) {
+      var map = new google.maps.Map(document.getElementById('map'), {
+        zoom: 2,
+        minZoom: 2,
+        center: new google.maps.LatLng(10, 15),
+        mapTypeId: 'terrain',
+        //disableDefaultUI: true,
+        mapTypeControl: false,
+        panControl: false,
+        //scaleControl: false,
+        scrollwheel: false,
+        streetViewControl: false,
+        //zoomControl: false,
```

Comment:
Yeah, I see the zoom control now, for some reason was missing it before :+1:

![](https://www.dropbox.com/s/07am120byj4axzr/Screenshot%202017-01-31%2008.51.07.png?dl=1)